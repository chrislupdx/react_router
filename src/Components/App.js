import React from 'react';
import {
  BrowserRouter as Router,
  Switch,
  Route
} from 'react-router-dom';
import Header from './Header';
import Color from './Color';
import Red from './Red';
import Blue from './Blue';
import Yellow from './Yellow';

export default function App() {
  return (
    <Router>
        <>
        <Header />
        <Switch>
          <Route path ="/red" component={Red} />
          <Route path ="/blue" component={Blue} />
          <Route path ="/yellow" component={Yellow} />
          <Route path="/:color" component={Color} />
        </Switch>
        </>
    </Router>
  );
}


