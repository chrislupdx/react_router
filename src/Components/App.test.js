import React from 'react';
import { shallow } from 'enzyme';
import App from './App';

describe('app', () => {
  it('renders', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
