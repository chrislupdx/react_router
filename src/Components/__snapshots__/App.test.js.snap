// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app renders 1`] = `
<BrowserRouter>
  <Header />
  <Switch>
    <Route
      component={[Function]}
      path="/red"
    />
    <Route
      component={[Function]}
      path="/blue"
    />
    <Route
      component={[Function]}
      path="/yellow"
    />
    <Route
      component={[Function]}
      path="/:color"
    />
  </Switch>
</BrowserRouter>
`;
